larq/optimizers_test.py

```diff
@@ -104,6 +104,24 @@ def test_bop_accuracy(self):
             test_kernels_are_binary=True,
         )
 
+    @pytest.mark.skipif(
+        utils.tf_1_14_or_newer() is False,
+        reason="tf.keras.optimizers.schedules only exists in TensorFlow 1.14",
+    )
+    def test_bop_tf_1_14_schedules(self):
+        _test_optimizer(
+            lq.optimizers.Bop(
+                threshold=tf.keras.optimizers.schedules.InverseTimeDecay(
+                    3.0, decay_steps=1.0, decay_rate=0.5
+                ),
+                gamma=tf.keras.optimizers.schedules.InverseTimeDecay(
+                    3.0, decay_steps=1.0, decay_rate=0.5
+                ),
+                fp_optimizer=tf.keras.optimizers.Adam(0.01),
+            ),
+            test_kernels_are_binary=True,
+        )
+
```

Comment:
is the default accuracy target (0.75) in _test_optimizer test conservative enough to be proper test for optimizer with this new hyperparameters?

Reply:
Accuracy is above 90% so it should be fine.